UncomplicatedCustomRoles/UncomplicatedCustomRoles.csproj

```diff
@@ -36,17 +38,14 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <DebugType>full</DebugType>
     <PlatformTarget>x64</PlatformTarget>
-    <LangVersion>11</LangVersion>
     <ErrorReport>prompt</ErrorReport>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|x64'">
     <OutputPath>bin\x64\Release\</OutputPath>
     <DefineConstants>TRACE</DefineConstants>
     <Optimize>true</Optimize>
     <DebugType>pdbonly</DebugType>
     <PlatformTarget>x64</PlatformTarget>
-    <LangVersion>11</LangVersion>

```

Comment:
why?

Reply:
The Lange version get moved.
https://github.com/UncomplicatedCustomServer/UncomplicatedCustomRoles/pull/20/files#diff-2e672f6a45c0ce56d40bd884bf5263c3c93fe1648a4dae598026165f629f0cc4R13